tritium-slf4j/src/main/java/com/palantir/tritium/event/log/LoggingInvocationEventHandler.java

```diff
@@ -192,9 +191,6 @@ static String generateMessagePattern(int argCount) {
             message.append("{}");
         }
         message.append(") took {}ms");
-        Preconditions.checkState(estimatedSize == message.length(),
-                "Incorrect estimated size for %s arguments, expected %s, was %s",
-                argCount, estimatedSize, message.length());
```

Comment:
this was the one potential hot path and this check was more of a perf optimization sanity check that is not needed